typeql/language/insert.feature

```diff
@@ -2772,3 +2770,33 @@ Parker";
       $x isa bird;
       $x iid V123;
       """
+
+  Scenario: datetime object can be inserted and queried correctly (and time-zone-naively) regardless of OS timezone
+    Given set time-zone is: Asia/Calcutta
+    Given connection close all sessions
+    Given connection open schema session for database: typedb
+    Given session opens transaction of type: write
+    Given typeql define
+    """
+    define test_date sub attribute, value datetime;
```

Comment:
`test-date`. Hyphens seem to be typeql convention.
And indenting the query string (as seen in the typeql insert step below).